src/main/java/org/truffleruby/RubyContext.java

```diff
@@ -122,8 +123,12 @@ public RubyContext(RubyLanguage language, TruffleLanguage.Env env) {
                     "this JVM does not have the Graal compiler - performance will be limited - see doc/user/using-graalvm.md");
         }
 
-        rubyHome = findRubyHome();
-        Log.LOGGER.config(() -> String.format("ruby home: %s", rubyHome));
+        try {
+            rubyHome = findRubyHome();
+        } catch (IOException e) {
+            throw new JavaException(e);
+        }
+        Log.config("ruby home: %s", rubyHome);
```

Comment:
Before:

```
$ jt run -J-Djava.util.logging.config.file=doc/samples/logging.properties -e 14
$ cat ruby.log 
...
  <class>org.truffleruby.RubyContext</class>
  <method>&lt;init&gt;</method>
  <thread>1</thread>
  <message>ruby home: /Users/chrisseaton/Documents/ruby/truffleruby-workspace/truffleruby</message>
...
```

After:

```
$ jt run -J-Djava.util.logging.config.file=doc/samples/logging.properties -e 14
$ cat ruby.log 
...
  <class>org.truffleruby.Log</class>
  <method>config</method>
  <thread>1</thread>
  <message>ruby home: /Users/chrisseaton/Documents/ruby/truffleruby-workspace/truffleruby</message>
...
```

The helpers in `Log` are only there for boundaries. You can write `if (LOGGER.isLoggable(Level.CONFIG)) {` inline in `RubyContext` if you want, but don't remove the useful information about which class and method a log entry is coming from.